README.md

```diff
@@ -173,6 +192,28 @@ Possible values of this variable:
 * `{state: absent}`: the ansible host name is unset in the insights-client config file and Host Based Inventory (HBI) is updated to use the system host name.
 * any other string value: the ansible host name is changed in Host Based Inventory (HBI).
 
+```yaml
+    rhc_inights:
+      baseurl: "cert-api.access.redhat.com:443/r/insights"
+```
+
+Configures the Base URL for the Insights API.
+Defaults to "cert-api.access.redhat.com:443/r/insights".
+
+```yaml
+    rhc_insights:
+      redaction_file: "/etc/insights-client/file-redaction.yaml"
```

Comment:
Are these file paths the defaults?  What are the reasons someone would want to change these from the default values?

Reply:
How will users provide the contents of file-redaction.yaml and file-content-redaction.yaml?

Reply:
Yes, these file paths are the defaults. To be honest I don't know why someone would like to change these. I thought when making parameters configurable with this role it might just include these as well.

On the question how to provide the content I can think of two options (for both the code is currently missing):

- Template these files using ˋansible.builtin.templateˋ
- Have them stored somewhere reachable for the controller to copy them to the managed node by using ˋansible.builtin.copyˋ

Using Jinja2 templates to create these files would be more flexible but also more complex.

Reply:
> Yes, these file paths are the defaults. To be honest I don't know why someone would like to change these. I thought when making parameters configurable with this role it might just include these as well.

Ok.  I think the more likely use case is someone wanting to customize the settings in file-redaction.yaml and file-content-redaction.yaml - not sure if @ptoscano had plans for that.

> 
> On the question how to provide the content I can think of two options (for both the code is currently missing):
> 
>     * Template these files using ˋansible.builtin.templateˋ
> 
>     * Have them stored somewhere reachable for the controller to copy them to the managed node by using ˋansible.builtin.copyˋ

* provide the contents inline.

A few of our roles do something similar - for example, the podman role allows the user to provide quadlet definitions either in a file (using `copy`), a template (using `template`), or inline - see https://github.com/linux-system-roles/podman/?tab=readme-ov-file#podman_quadlet_specs

> 
> 
> Using Jinja2 templates to create these files would be more flexible but also more complex.

Yes. and we can support one of the methods at first then support the others later.  This is why I think we need to identify the use cases (perhaps @ptoscano has already done this).